en/reference/xml-reference.rst

```diff
@@ -0,0 +1,67 @@
+XML Mapping Reference
+=====================
+
+This chapter gives a brief overview of the XML mapping by example. In general
```

Comment:
I see it as a single phrase and when speaking it there would be no pause that is associated with a comma. Though tbh I am not sure what the exact gramatical rules are here.